virtual-server/examples/python/kubevirtclient.py

```diff
@@ -242,7 +242,12 @@ def start(self, namespace, name, **kwargs):
         """
         kwargs['_return_http_data_only'] = True
         version = self.version(namespace, name)
-        return self.kubevirt_api(namespace, name, 'start', version=version, **kwargs)
+        while True:
```

Comment:
After the stop command, sometimes VMI is still running and returns 409.

Reply:
Only really if the VM is quickly started/stopped. Good check nonetheless